modules/toxiproxy/src/main/java/org/testcontainers/containers/ToxiproxyContainer.java

```diff
@@ -86,6 +86,7 @@ public int getControlPort() {
      * @param port port number on the target service that should be proxied
      * @return a {@link ContainerProxy} instance
      */
+    @Deprecated
```

Comment:
Should we add to the Javadoc the deprecation comment, to use the Toxiproxy client directly instead?